test/views/VatReturnDetailsViewSpec.scala

```diff
@@ -558,18 +565,54 @@ class VatReturnDetailsViewSpec extends ViewBaseSpec {
       showReturnsBreadcrumb = true,
       currentYear,
       hasFlatRateScheme = true,
+      isOptOutMtdVatUser = false,
       isHybridUser = false
     )
 
-    lazy val view = views.html.returns.vatReturnDetails(vatReturnViewModel)
-    lazy implicit val document: Document = Jsoup.parse(view.body)
 
-    "have the pageHeading of 'Final return'" in {
-      elementText(Selectors.pageHeading) shouldBe "Submitted returns Final return"
-    }
+    "Rendering the VAT return details page when the user is opted out" should {
+
+      val vatReturnViewModel = VatReturnViewModel(
+        None,
+        LocalDate.parse("2017-01-01"),
+        LocalDate.parse("2017-03-31"),
+        LocalDate.parse("2017-04-06"),
+        1000.00,
+        LocalDate.parse("2017-04-08"),
+        VatReturnDetails(
+          VatReturn(
+            "9999",
+            1297,
+            5755,
+            7052,
+            5732,
+            1000,
+            77656,
+            765765,
+            55454,
+            545645
+          ),
+          moneyOwed = true,
+          oweHmrc = Some(true),
+          None
+        ),
+        showReturnsBreadcrumb = true,
+        currentYear,
+        hasFlatRateScheme = true,
+        isOptOutMtdVatUser = true,
+        isHybridUser = false
+      )
 
-    "have the breadcrumb heading of 'Final return" in {
-      elementText(Selectors.currentPage) shouldBe "Final return"
+      lazy val view = views.html.returns.vatReturnDetails(vatReturnViewModel)
+      lazy implicit val document: Document = Jsoup.parse(view.body)
+
+      "have the pageHeading of 'Final return'" in {
```

Comment:
We don't need these two checks - just need to check the content of the progressive disclosure